src/mbgl/style/expression/at.cpp

```diff
@@ -19,10 +19,16 @@ EvaluationResult At::evaluate(const EvaluationContext& params) const {
     const auto i = evaluatedIndex->get<double>();
     const auto inputArray = evaluatedInput->get<std::vector<Value>>();
     
-    if (i < 0 || i >= inputArray.size()) {
+    if (i < 0) {
+        return EvaluationError {
+            "Array index out of bounds: " + stringify(i) + " < 0."
```

Comment:
Is there a reason we're mixing `stringify` and `util::toString`?

Reply:
No, just an oversight, I think.  Will consistentize before merging.